examples/ListDir.hs

```diff
@@ -1,20 +1,22 @@
+{-# LANGUAGE FlexibleContexts #-}
 module Main (main) where
 
 import Data.Bifunctor (bimap)
 import Data.Function ((&))
-import Streamly.Prelude (SerialT)
+import Streamly.Data.Stream (Stream)
 import System.IO (stdout, hSetBuffering, BufferMode(LineBuffering))
 
-import qualified Streamly.Prelude as Stream
+import qualified Streamly.Data.Fold as Fold
 import qualified Streamly.Internal.Data.Stream as Stream
-       (iterateMapLeftsWith)
-import qualified Streamly.Internal.FileSystem.Dir as Dir (toEither)
+       (iterateMapLeftsWith, fold, fromPure)
```

Comment:
`fold` and `fromPure` are not internal, they can be imported from released module.